### PR TITLE
Update DNS information

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # GMT 中文社区维护指南
 
+## DNS信息
```
Comment (core-man) [inline README.md]:
```suggestion
## DNS 信息
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+DNS使用 [腾讯云](https://cloud.tencent.com/) 进行解析。
```
Comment (core-man) [inline README.md]:
```suggestion
DNS 使用[腾讯云](https://cloud.tencent.com/)进行解析。
```

File: README.md [hunk 3]
```diff
@@ -19,4 +44,3 @@
 
 [seisman]: http://github.com/seisman
 [GoDaddy]: https://www.godaddy.com
-[DNSPod]: https://www.dnspod.cn
```
Comment (core-man) [inline README.md:42]: `2016-03-20: [@seisman][seisman] 将 DNS 服务器转向 [DNSPod][DNSPod] ，并设置 A 记录指向 GitHub`

There is an [DNSPod] in the history.

File: README.md (hunk 1 above)
Comment (core-man) [inline README.md]: Put DNS after GMT-China history?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+
+- 账户所有者: [@seisman](seisman)
+
+具体操作：
+
+1. 到 Godaddy 的“我的域名”->“域名设置”下，找到“管理DNS”
+2. 修改域名服务器为 `f1g1ns1.dnspod.net` 和 `f1g1ns2.dnspod.net`
+3. 前往腾讯云，添加域名 `gmt-china.org`
+4. 添加 `A` 记录，指定 GitHub Pages IP 地址 `185.199.108.153`
+5. 增加CNAME 记录，二级域名 `docs.gmt-china.org` 指向 `gmt-china.github.io`
```
Comment (core-man) [inline README.md]: Add a figure like https://github.com/seismo-learn/maintenance#dns%E4%BF%A1%E6%81%AF?

Comment (core-man) [inline README.md]: We have `gmt-china.github.io`?
```suggestion
5. 增加 CNAME 记录，二级域名 `docs.gmt-china.org` 指向 `gmt-china.github.io`
```

File: README.md (hunk 3 above)
Comment (seisman) [reply README.md:42]: Dnspod被腾讯收购了

Comment (core-man) [reply README.md:42]: 我的意思是发展历史中有个 [DSNPod]链接，你删除了 [DNSPod]: https://www.dnspod.cn。链接就无效了

File: README.md [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+
+- 账户所有者: [@seisman](seisman)
+
+具体操作：
+
+1. 到 Godaddy 的“我的域名”->“域名设置”下，找到“管理DNS”
+2. 修改域名服务器为 `f1g1ns1.dnspod.net` 和 `f1g1ns2.dnspod.net`
+3. 前往腾讯云，添加域名 `gmt-china.org`
+4. 添加 `A` 记录，指定 GitHub Pages IP 地址 `185.199.108.153`
+5. 增加 CNAME 记录，二级域名 `docs.gmt-china.org` 指向 `gmt-china.github.io`
```
Comment (core-man) [inline README.md:49]: `gmt-china.github.io` 存在吗？

Comment (seisman) [reply README.md:49]: 与是否存在无关，`docs` 的 CNAME 指向 gmt-china.github.io，然后访问 docs.gmt-china.org 的时候访问请求就会发送到 github，github 会从自己的记录里找到对应的 CNAME 配置，然后返回 GMT_Docs 的 gh-branches 分支。

File: README.md [hunk 6]
```diff
@@ -32,6 +32,35 @@
...
+6. 为使用腾讯企业邮箱，需增加如下记录
+
+   ```
+   @  MX   mxbiz1.qq.com  5
+   @  MX   mxbiz2.qq.com  10
+   ```
+
+DNS 记录如下图所示:
+
+[](https://user-images.githubusercontent.com/3974108/123645067-fd33fe80-d7f3-11eb-8aad-05b898981e52.png)
```
Comment (core-man) [inline README.md]:
```suggestion
![](https://user-images.githubusercontent.com/3974108/123645067-fd33fe80-d7f3-11eb-8aad-05b898981e52.png)
```

File: README.md [hunk 7]
```diff
@@ -32,6 +32,35 @@
...
+   @  MX   mxbiz2.qq.com  10
+   ```
+
+DNS 记录如下图所示:
+
+![](https://user-images.githubusercontent.com/3974108/123645067-fd33fe80-d7f3-11eb-8aad-05b898981e52.png)
+
+生成静态网站并将网站托管到 GitHub Pages 时，还需要注意如下两点：
+
+1. 要有一个空文件 `.nojekyll` 以免 GitHub 使用 Jekyll 对网站再次渲染
```
Comment (core-man) [inline README.md:63]: We don't have `.nojekall` files in docs or gmt-china.org.

File: README.md [hunk 8]
```diff
@@ -32,6 +32,35 @@
...
+2. 要有一个 `CNAME` 文件，其中包括当前网站的域名，例如 `gmt-china.org`
```
Comment (core-man) [inline README.md:64]: We don't have a `CNAME` file in gmt-china.org, but one in docs.

Comment (seisman) [reply README.md:64]: We have it in the `static` directory.

File: README.md (hunk 7 above)
Comment (seisman) [reply README.md:63]: https://github.com/gmt-china/GMT_docs/tree/gh-pages
https://github.com/gmt-china/gmt-china.org/tree/gh-pages

Comment (core-man) [reply README.md:63]: docs use `deploy` to generate `.nojekall`, but how gmt-china.org create this file? I guess hugo and wow will generate it.

Comment (seisman) [reply README.md:63]: Maybe.

File: README.md [hunk 9]
```diff
@@ -32,11 +34,41 @@
...
+
 ## 邮箱账户
 
 中文社区目前使用[腾讯企业邮](https://exmail.qq.com/)创建了专属邮箱账户：
 `admin@gmt-china.org`，账户所有者 [@seisman][seisman]。
```
Comment (core-man) [inline README.md]:
```suggestion
中文社区目前使用[腾讯企业邮](https://exmail.qq.com/)创建了专属邮箱账户 admin@gmt-china.org，账户所有者 [@seisman][seisman]。
```

**Review by core-man — APPROVED** [review]

**Pull request merged**